reporter/src/funTest/assets/evaluated-model-reporter-test-expected-output.json

```diff
@@ -92,6 +92,7 @@
     "start_time" : "1970-01-01T00:00:00Z",
     "end_time" : "1970-01-01T00:00:00Z",
     "file_count" : 0,
+    "package_verification_code" : "e824b43fc02fa979e1377164ba26a739790c281b",
```

Comment:
Nit: If the package verification is fake, does not make sense to highlight that e.g. by only using zeros here?

Reply:
Done.